models/Movie.model.js

```diff
@@ -0,0 +1,47 @@
+const { Schema, model } = require("mongoose");
+
+const movieSchema = new Schema(
+  {
+    title: {
+      type: String,
+      require: true
+    },
+    plot: {
+      type: String,
+    },
+    genre: {
+      type: String,
+    },
+    year: {
+      type: String,
+    },
+    director: {
+      type: String,
+    },
+    cast: {
+      type: String,
+    },
```

Comment:
wish: That was written wrong in the tasks description. I think we should use here an array of Strings

```suggestion
cast: [ String ]
```